code/CO2_monitor.ino

```diff
@@ -334,6 +369,20 @@ void loop(){
       co2_web         = co2_new;
       temperature_web = temperature_new;
       humidity_web    = humidity_new;
+#if WIFI_MQTT
+      client.connect(deviceName);
+#if SEND_VCC
+      sprintf(buf, "%d", vdd);
+      client.publish("esp-co2/co2/vcc", buf );
+#endif
+      client.publish("esp-co2/co2/hostname", deviceName );
+      sprintf(buf, "%6.2f", co2_web);
+      client.publish("esp-co2/co2/co2", buf );
+      sprintf(buf, "%6.2f", temperature_web);
+      client.publish("esp-co2/co2/temp", buf );
+      sprintf(buf, "%6.2f", humidity_web);
+      client.publish("esp-co2/co2/hum", buf );
```

Comment:
Following my comment/suggestion from above this might need to be changed to

```
      mqttClient.publish("esp-co2/co2/hostname", deviceName );
      sprintf(mqttMessage, "%6.2f", co2_web);
      mqttClient.publish("esp-co2/co2/co2", buf );
      sprintf(mqttMessage, "%6.2f", temperature_web);
      mqttClient.publish("esp-co2/co2/temp", buf );
      sprintf(mqttMessage, "%6.2f", humidity_web);
      mqttClient.publish("esp-co2/co2/hum", buf );
```